docs-2.0/20.appendix/0.FAQ.md

```diff
@@ -357,31 +357,54 @@ $ ./nebula-graphd --version
 
   执行`rpm -qa |grep nebula`即可查看版本。
 
-{{ ent.ent_begin }}
-### 如何扩缩容（仅限企业版）
+### 如何扩缩容    
 
-- 使用悦数运维监控，在可视化页面对 graphd 和 storaged 进行快速扩缩容，详情参见[集群操作-扩缩容](../nebula-dashboard-ent/4.cluster-operator/operator/scale.md)。
-- 使用 NebulaGraph Operator 扩缩容集群，详情参见[使用 Kubectl 部署{{nebula.name}}集群](../nebula-operator/3.deploy-nebula-graph-cluster/3.1create-cluster-with-kubectl.md)和[使用 Helm 部署{{nebula.name}}集群](../nebula-operator/3.deploy-nebula-graph-cluster/3.2create-cluster-with-helm.md)。
+{{comm.comm_begin}}
+!!! warning  
+   
+    集群扩缩容功能未正式在社区版中发布。以下涉及`BALANCE DATA REMOVE`和`BALANCE DATA`的操作在社区版中均为实验性功能，功能未稳定。如需使用，请先做好数据备份，并且在 [Graph 配置文件](../5.configurations-and-logs/1.configurations/3.graph-config.md)中将`enable_experimental_feature`设置为`true`。
 
-{{nebula.name}} {{ nebula.release }} 未提供运维命令以实现自动扩缩容，参考以下步骤：
+{{comm.comm_end}}
 
-- metad 的扩容和缩容： metad 不支持自动扩缩容。
+#### 增加或减少 Meta、Graph、Storage 节点的数量
+
+- {{nebula.name}} {{ nebula.release }} 未提供运维命令以实现自动扩缩容，但可参考以下步骤实现手动扩缩容：
+
+  - Meta 的扩容和缩容： Meta 不支持自动扩缩容。
+
+    !!! note
+    
+          用户可以使用[脚本工具](https://github.com/vesoft-inc/nebula/blob/master/scripts/meta-transfer-tools.sh) 迁移 Meta 服务，但是需要自行修改 Graph 服务和 Storage 服务的配置文件中的 Meta 设置。
+
+  - Graph 的缩容：将该 Graph 的 IP 从 client 的代码中移除，关闭该 Graph 进程。
+
+  - Graph 的扩容：在新机器上准备 Graph 二进制文件和配置文件，在配置文件中修改或增加已在运行的 Meta 地址，启动 Graph 进程。
+
+  - Storage 的缩容：执行`BALANCE DATA REMOVE <ip:port>`和`DROP HOSTS <ip:port>`。完成后关闭 Storage 进程。
+
+  - Storage 的扩容：在新机器上准备 Storage 二进制文件和配置文件，在配置文件中修改或增加已在运行的 Meta 地址，然后注册 Storage 到 Meta 并启动 Storage 进程。详情参见[注册 Storage 服务](../2.quick-start/3.quick-start-on-premise/3.1add-storage-hosts.md)。
+
+    Storage 扩缩容之后，还需要运行`BALANCE DATA`和`BALANCE LEADER`命令均衡数据和 Leader 分布。
+
+ {{ent.ent_begin}}
+- 使用悦数运维监控，在可视化页面对 Graph 和 Storage 进行快速扩缩容，详情参见[集群操作-扩缩容](../nebula-dashboard-ent/4.cluster-operator/operator/scale.md)。
+
+- 使用 NebulaGraph Operator 扩缩容集群，详情参见[使用 Kubectl 部署{{nebula.name}}集群](../nebula-operator/3.deploy-nebula-graph-cluster/3.1create-cluster-with-kubectl.md)和[使用 Helm 部署{{nebula.name}}集群](../nebula-operator/3.deploy-nebula-graph-cluster/3.2create-cluster-with-helm.md)。
+ {{ent.ent_end}}
 
-  !!! note
-  
-         用户可以使用[脚本工具](https://github.com/vesoft-inc/nebula/blob/master/scripts/meta-transfer-tools.sh) 迁移 meta 服务，但是需要自行修改 Graph 服务和 Storage 服务的配置文件中的 Meta 设置。
 
-- graphd 的缩容：将该 graphd 的 ip 从 client 的代码中移除，关闭该 graphd 进程。
+#### 在 Storage 节点中增加磁盘
 
-- graphd 的扩容：在新机器上准备 graphd 二进制文件和配置文件，在配置文件中修改或增加已在运行的 metad 地址，启动 graphd 进程。
+目前，Storage 不支持动态识别新的磁盘。可以通过以下步骤在分布式集群的 Storage 节点中增加磁盘{{comm.comm_begin}}（不支持对单个 Storage 节点的集群增加磁盘）{{comm.comm_end}}：
 
-- storaged 的缩容：详情参见[缩容命令](../8.service-tuning/load-balance.md#_5)。完成后关闭 storaged 进程。
+1. 执行`BALANCE DATA REMOVE <ip:port>`将待增加磁盘的 Storage 节点迁到其他 Storage 节点中。
```

Comment:
We may add notice here to clarify that REMOVE required us to have other storage hosts that can fulfill the replica factor, i.e. if there are space with 3 replication but we only have 3 storage hosts in all, the remove cannot be done :D